copy.go

```diff
@@ -224,6 +212,30 @@ func dcopy(srcdir, destdir string, info os.FileInfo, opt Options) (err error) {
 	return
 }
 
+// dread reads the named directory,
+// it regardless if it's a filesystem
+// and returns list of directory entries.
+func dread(srcdir string, fsys fs.FS) ([]fs.DirEntry, error) {
+	if fsys != nil {
+		return fs.ReadDir(fsys, srcdir)
+	}
+	entries, err := os.ReadDir(srcdir)
+	if err != nil {
+		if !os.IsNotExist(err) {
```

Comment:
Even in `fs.FS`, let's ignore `os.IsNotExist`, then this function can be more readable.